api/tests/certification/session/acceptance/application/update-cpf-import-status-controller_test.js

```diff
@@ -0,0 +1,47 @@
+import {
+  expect,
+  generateValidRequestAuthorizationHeader,
+  insertUserWithRoleSuperAdmin,
+} from '../../../../test-helper.js';
+import { createServer } from '../../../../../server.js';
+
+describe('Acceptance | Controller | Session | update-cpf-import-status-controller', function () {
+  let server;
+
+  beforeEach(async function () {
+    server = await createServer();
+  });
+
+  describe('POST /api/cpf/receipts', function () {
+    it('should return an ACCEPTED (202) status', async function () {
+      // given
+      const superAdmin = await insertUserWithRoleSuperAdmin();
+      const options = {
+        method: 'POST',
+        url: '/api/cpf/receipts',
+        headers: { authorization: generateValidRequestAuthorizationHeader(superAdmin.id) },
+      };
+
+      // when
+      const response = await server.inject(options);
+
+      // then
+      expect(response.statusCode).to.equal(202);
+    });
+
+    it('should respond with a 403 when user cant access the session', async function () {
```

Comment:
**Suggestion :** un petit module 😄 ? ( `module('when user can't access the session', function (`) )

Reply:
J'ai ajouté un describe (module étant pour les front).